languages-and-frameworks/wordpress.html.markerb

```diff
@@ -0,0 +1,160 @@
+
+Before deploying, we'll need to configure a couple of things. In your newly created fly.toml file, change the `internal_port` under `[http_service]` from 8080 to **80**. This setting lets Fly know on what port your app will respond, which is 80 in our case.
+
+After that, set up these environment variables by adding this to the end of your `fly.toml`:
+
+```toml
+[env]
+  WORDPRESS_DB_NAME= "wordpress_db"
+  WORDPRESS_DB_HOST= "<name-of-your-mysql-app>.internal"
+  WORDPRESS_CONFIG_EXTRA = "define( 'WP_HOME', 'https://<name-of-your-wordpress-app>.fly.dev' );define( 'WP_SITEURL', 'https://<name-of-your-wordpress-app>.fly.dev');"
+```
+
+The `WORDPRESS_DB_NAME` and `WORDPRESS_DB_HOST` are needed for the wordpress application to connect to the database. The `WORDPRESS_CONFIG_EXTRA` makes sure that the application serves everything from the `https://...` address.
+
+Here's how the total fly.toml should look now:
+```toml
+app = 'fly-wordpress'
+primary_region = 'fra'
+
+[build]
+  image = 'wordpress'
+
+[http_service]
+  internal_port = 80 # changed from 8080 to 80
+  force_https = true
+  auto_stop_machines = true
+  auto_start_machines = true
+  min_machines_running = 0
+  processes = ['app']
+
+[[vm]]
+  memory = '1gb'
+  cpu_kind = 'shared'
+  cpus = 1
+
+# Add these environment variables
+[env]
+WORDPRESS_DB_NAME= "wordpress_db"
+WORDPRESS_DB_HOST= "db-wordpress.internal"
+WORDPRESS_CONFIG_EXTRA = "define( 'WP_HOME', 'https://fly-wordpress.fly.dev' );define( 'WP_SITEURL', 'https://fly-wordpress.fly.dev');"
+```
+
+Next, we need to set up the `WORDPRESS_DB_USER` and `WORDPRESS_DB_PASSWORD` variables. Since these are sensitive, let's use the [`fly secrets`](/docs/reference/secrets/#set-secrets) command:
+
+```cmd
+flyctl secrets set WORDPRESS_DB_USER=wordpress_user WORDPRESS_DB_PASSWORD=wordpress_password
+```
+```output
+Secrets are staged for the first deployment
+```
+
+You can list the secrets you've already set using `fly secrets list`. For example:
+
+```cmd
+fly secrets list
+```
+```
+NAME                    DIGEST                  CREATED AT
+WORDPRESS_DB_PASSWORD   92d2791e3f7ebd71        1m20s ago
+WORDPRESS_DB_USER       a2b707ef6e858717        1m20s ago
+```
+
+The values aren't displayed, since they're secret!
+
+### Deploy your app
+
+The last step is to deploy the app with the deploy command:
+
+```cmd
+flyctl deploy
+```
+```output
+==> Verifying app config
+Validating /Users/name/test/my-app-name/fly.toml
+✓ Configuration is valid
+--> Verified app config
+==> Building image
+Searching for image 'wordpress' remotely...
+image found: img_rlw04nnx6zzj4z2y
+
+
+Watch your deployment at https://fly.io/apps/my-app-name/monitoring
+
+-------
+Updating existing machines in 'my-app-name' with rolling strategy
+
+-------
+ ✔ [1/2] Machine 918543b477de83 [app] update succeeded
+ ✔ [2/2] Machine e28697ce6d3986 [app] update succeeded
+-------
+
+Visit your newly deployed app at https://my-app-name.fly.dev/
+```
+
+### Open your app
```

Comment:
Do you think a lot of wordpress users would be using a custom domain? We can [link to that](https://fly.io/docs/networking/custom-domain/) or even include a quick version here?